### DISCUSSION TITLE
Long range correction and wigner phonons

### DISCUSSION THREAD

**Discussion by akentphonon** [phoebe-team/phoebe#146] [Q&A] [answered]
Dear developers @cepellotti @jcoulter12,

Thank you for the great code - I find it to be easy to install and super fast for doing phonon transport calculations!

I have a couple of (perhaps naive) quires about phonon transport calculations in Phoebe-

1. Is it possible to include long-range corrections when calculating phonons and phonon Transport - specifically when using IFCs from phonopy/phono3py?

2. About the results written in "wigner_phonon_thermal_cond.json" - I suppose these are total results for \kappa_particle + \kappa_coherence. Is that correct? If want to know contribution of only  \kappa_coherence, can I simply compare the results of "wigner_phonon_thermal_cond.json" with "omini_phonon_thermal_cond.json" – and the difference will be the contribution of \kappa_coherence?

Looking forward to your valuable insights.

Thanks a lot for your time :)
Alex

**Comment by jcoulter12** [comment, marked as answer]
Hi Alex, 

It's great to hear you're enjoying Phoebe!
We're released a few minor changes to develop branch in response to your questions. 

For your first question, this is indeed possible if you prepare the phononpy BORN file, as indicated here: 
https://phonopy.github.io/phonopy/input-files.html#born-optional

 and then set the new Phoebe input variable (also added now to the documentation of inputs): 

`phonopyBORNFileName = "BORN" `

You should be able to add this correction to the IFC2s and see the optical band splitting. Let us know if this doesn't go as expected for any reason. I've tested it on a few systems, but you never know. 

On your second question, thanks for the feedback -- I can see how the Wigner contribution output is a little bit confusing. We'll likely update this soon to a better format. 

Fortunately, if you're interested only in the phonon thermal conductivity, the Wigner correction is simply additive. There's actually a short note about this in the "Output" section of the phonon transport tutorial: 

> Note: though we output the Wigner correction + the RTA thermal conductivity, the Wigner correction is additive, and can be applied to any solver’s output thermal conductivity. Therfore, if you want Wigner+iterative thermal conductivity, you should take \kappa_{Wigner+RTA} - \kappa_{RTA} + \kappa_{Omini}.

Let us know if you have any more questions or issues,  
Jenny

**Reply by akentphonon** [reply]
Thanks a lot Jenny. The Long range corrections works perfectly. :)
